mediathread/assetmgr/models.py

```diff
@@ -270,6 +270,12 @@ def global_annotation(self, user, auto_create=True):
 
     def media_type(self):
         label = 'video'
+
+        try:
```

Comment:
Is there a situation where primary does not exist?

Reply:
During testing, with the new is_global_annotation() change, some of the reports tests fail with Source.DoesNotExist. Debugging this was pretty confusing - we can possibly solve this another way, or postpone the global annotation changes altogether by just using the hardcoded ranges in the new PDF annotation structure.

Reply:
I like your suggestion of just using hard-coded values in the ranges. To me, that feels cleaner then adding additional exception code. What's your feeling?

Reply:
Sure, that's fine - i mean that's what we're currently doing for image annotations. That's what I was trying to get rid of in the first place, but I can just do that for now if we don't want to spend time re-thinking global annotations yet.

Reply:
Yeah - removing one workaround for images/pdf seems simpler than removing two. That said, I'm fine talking through how we can remove global annotations now. No reason to put that off.